shell/components/Questions/__tests__/Boolean.test.ts

```diff
@@ -38,7 +37,7 @@ describe.skip('(Vue3 Skip) the Boolean Component', () => {
     expect(labelFields.at(0).text()).toBe('var_name');
   });
 
-  it('description is present', () => {
+  it.skip('description is present', () => {
```

Comment:
Is this intentional or a mistake? 👀 
I do not see in the list of missing tests.

Reply:
Not intentional - updated now